### PR TITLE
Call system messages follow-up

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Design review changes to `CallCell`s.
* Update `PingCell`.
* Do not show burst timestamp for call cells.

### CODE REVIEW

**Comment by mikeger** [comment]
@daehn 🐗 Conflict

**Comment by daehn** [comment]
@mikeger updated

**Comment by daehn** [comment]
Need to check one flaky test

**Comment by daehn** [comment]
@mikeger fixed the flaky one

File: Wire-iOS Tests/CallSystemMessageTests.swift [hunk 1]
```diff
@@ -73,10 +73,13 @@ class CallSystemMessageTests: CoreDataSnapshotTestCase {
     }
 
     private func systemMessage(missed: Bool, in conversation: ZMConversation, from user: ZMUser) -> ZMSystemMessage {
+        let date = Date(timeIntervalSince1970: 123456879)
```
Comment (mikeger) [inline Wire-iOS Tests/CallSystemMessageTests.swift]: Was a good year!

File: Wire-iOS Tests/CallSystemMessageTests.swift [hunk 2]
```diff
@@ -73,10 +73,13 @@ class CallSystemMessageTests: CoreDataSnapshotTestCase {
...
         if missed {
-            return conversation.appendMissedCallMessage(fromUser: user, at: Date(timeIntervalSince1970: 123456879))
+            return conversation.appendMissedCallMessage(fromUser: user, at: date)
         } else {
-            return conversation.appendPerformedCallMessage(with: 102, caller: user)
+            let message = conversation.appendPerformedCallMessage(with: 102, caller: user)
```
Comment (mikeger) [inline Wire-iOS Tests/CallSystemMessageTests.swift]: `with: 102` is not too informative for the method label, would be better to have like `duration: 102`

**Pull request merged**